invenio_oauthclient/contrib/keycloak/helpers.py

```diff
@@ -53,7 +53,45 @@ def get_public_key(remote, realm_url):
     return certs_resp["public_key"]
 
 
-def get_user_info(remote, resp_token, fallback_to_endpoint=True, options=dict()):
```

Comment:
`fallback_to_endpoint=True, options=dict()` were actually unreachable, since both the place where this function is called and the corresponding handler does not hage `*args, **kwargs` (and therefore does not forward them neither).

Reply:
these arguments are passed in the tests: https://github.com/inveniosoftware/invenio-oauthclient/blob/master/tests/test_contrib_keycloak.py#L381

though the `options` there look like they could be replaced via tweaking `app.config` a bit
and the fallback endpoint shouldn't be called in the test anyway (`fallback_to_endpoint=False`)... could be mocked with `httpretty` to return a failure to make sure it's not called